shuup/reporting/reporter/pdf_export.py

```diff
@@ -0,0 +1,22 @@
+# -- encoding: UTF-8 --
```

Comment:
Missing license header and seems to be duplicate code with `shuup.order_printouts.pdf_export` Maybe this should be somewhere under `shuup.utils.pdf`?